activities/TankOp.activity/index.html

```diff
@@ -42,6 +49,7 @@
 	  <button class="toolbutton" id="activity-button" title="Tank Operation"></button>
 	  <p class="title">TANK OPERATION</p>
 	  <button class="toolbutton pull-right" id="stop-button" title="Stop"></button>
+	  <button class="toolbutton pull-right" id="help-button" title="Tutorial"></button>
```

Comment:
The help button should be to the left of fullscreen button

Reply:
Order Corrected.